src/gql/queries/get-mainline-commits.graphql

```diff
@@ -1,4 +1,7 @@
-query MainlineCommits($options: MainlineCommitsOptions!) {
+query MainlineCommits(
+  $options: MainlineCommitsOptions!
+  $taskStatusCountsOptions: BuildVariantOptions!
```

Comment:
can we name this `$buildVariantOptions` Since we will reuse it for the buildVariant filters